README.rst

```diff
@@ -0,0 +1,105 @@
+image here
+image target here
+
+image here
+image target here
+
+
+PDFmorph
+========================================================================
+add description, sample below
```

Comment:
please add a Fixme to anything that needs to be returned to.  This makes them easier to find and we don't accidentally release code with place-holders.

Reply:
done

Reply:
👍